ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/tasks/workload.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: Determine cluster wildcard domain
+  kubernetes.core.k8s_info:
+    api_version: operator.openshift.io/v1
+    kind: IngressController
+    name: default
+    namespace: openshift-ingress-operator
+  register: r_ingress_controller
+
+- name: Save cluster apps domain variable
+  ansible.builtin.set_fact:
+    _ocp4_workload_cloud_native_robot_shared_apps_domain: "{{ r_ingress_controller.resources[0].status.domain }}"
```

Comment:
The `host-ocp4-set-facts` role (https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles/host-ocp4-set-facts) already sets the fact `openshift_cluster_ingress_domain` which can be used here.

Reply:
Thanks, useless code `_ocp4_workload_cloud_native_robot_shared_apps_domain` is not in use. Removed.